README.md

```diff
@@ -11,22 +11,40 @@ Bash-it provides a solid framework for using, developing and maintaining shell s
 If you're using the _Bourne Again Shell_ (Bash) on a regular basis and have been looking for an easy way on how to keep all of these nice little scripts and aliases under control, then Bash-it is for you!
 Stop polluting your `~/bin` directory and your `.bashrc` file, fork/clone Bash-it and start hacking away.
 
+- [Contributing](#contributing)
+- [Installation](#installation)
+  - [Install Options](#install-options)
+  - [via Docker](#install-using-docker)
+  - [Updating](#updating)
+- [Help](#help-screens)
+- [Search](#search)
+  - [Syntax](#syntax)
+  - [Searching with Negations](#searching-with-negations)
+  - [Using Search to Enable or Disable Components](#using-search-to-enable-or-disable-components)
+  - [Disabling ASCII Color](#disabling-ascii-color)
+- [Custom scripts, aliases, themes, and functions](#custom-scripts,-aliases,-themes,-and-functions)
+- [Themes](#themes)
+- [Uninstalling](#uninstalling)
+- [Misc](#misc)
+- [Help Out](#help-out)
+- [Contributors](#contributors)
+
 ## Contributing
 
 Please take a look at the [Contribution Guidelines](CONTRIBUTING.md) before reporting a bug or providing a new feature.
 
 The [Development Guidelines](DEVELOPMENT.md) have more information on some of the internal workings of Bash-it,
 please feel free to read through this page if you're interested in how Bash-it loads its components.
 
-## Install
+## Installation
 
 1. Check out a clone of this repo to a location of your choice, such as
    `git clone --depth=1 https://github.com/Bash-it/bash-it.git ~/.bash_it`
 2. Run `~/.bash_it/install.sh` (it automatically backs up your `~/.bash_profile` or `~/.bashrc`, depending on your OS)
 3. Edit your modified config (`~/.bash_profile` or `~/.bashrc`) file in order to customize Bash-it.
 4. Check out available aliases, completions, and plugins and enable the ones you want to use (see the next section for more details).
 
-### INSTALL OPTIONS
+#### INSTALL OPTIONS
```

Comment:
Since we're changing this, wanna move it to regular title-case too?

I'm also curious why the move from `<h3>` to `<h4>`?

Reply:
Well, it's more uniform for all subheadings to have the same size, I have no problem if they are `<h3>` or `<h4>`. Just that they are uniform

Reply:
My main goal is of adding uniform headers

Reply:
Gotcha – love the idea of having them all be uniform. Let's have them go `##` to `###` so we keep the semantic nature if that's ok with you.

Reply:
Yes, I like that idea!

I must say that I'm not too thrilled about the hand-written table of contents, though. Having a TOC is nice, but I'm afraid that it will go out of sync quickly if people don't update it. If possible, I'd like to have a way to generate the TOC, e.g. something like [this](https://github.com/jonschlinkert/markdown-toc), and then a documented way on how to use it...